testsuite/MDAnalysisTests/test_api.py

```diff
@@ -49,3 +55,20 @@ def test_ResidueGroup():
 
 def test_SegmentGroup():
     assert mda.SegmentGroup is mda.core.groups.SegmentGroup
+
+def test_all_imports():
+    os.chdir(os.path.dirname(mda.__file__))
+
+    for root, dirs, files in os.walk("."):
+        if "__init__.py" in files:
+            submodule = ".".join(PurePath(root).parts)
+            submodule = "."*(len(submodule) > 0) + submodule
```

Comment:
You could extract the code from the beginning of this function up to here in a separate function that generate the list (or a generator? would that work with pytest?) to pass to `pytest.mark.parametrize` so that each module is tested separately. This way you can fix all the broken ones rather than fix one, rerun tests, fix the next broken one... This is just me becoming greedy here, though.

Reply:
Yes, good idea to split this into individual tests. This is also easy to implement. No greed needed :)